src/main/java/com/netflix/imflibrary/st0429_8/PackingList.java

```diff
@@ -67,14 +66,35 @@
 {
     private static final Logger logger = LoggerFactory.getLogger(PackingList.class);
 
-    private static final String pkl_schema_path = "org/smpte_ra/schemas/st0429_8_2007/PKL/packingList_schema.xsd";
     private static final String xmldsig_core_schema_path = "org/w3/_2000_09/xmldsig/xmldsig-core-schema.xsd";
-    public static final List<String> supportedPKLSchemaURIs = Collections.unmodifiableList(new ArrayList<String>(){{ add("http://www.smpte-ra.org/schemas/429-8/2007/PKL");}});
+    public static final List<String> supportedPKLNamespaces = Collections.unmodifiableList(new ArrayList<String>(){{ add("http://www.smpte-ra.org/schemas/429-8/2007/PKL");}});
 
-    private final PackingListType packingListType;
     private final UUID uuid;
+    private final JAXBElement packingListTypeJAXBElement;
+    private final PKLSchema pklSchema;
     private final List<Asset> assetList = new ArrayList<>();
 
+    private static class PKLSchema {
+        private final String pklSchemaPath;
+        private final String pklContext;
+
+        private PKLSchema(String pklSchemaPath, String pklContext){
+            this.pklSchemaPath = pklSchemaPath;
+            this.pklContext = pklContext;
+        }
+
+        private String getPKLSchemaPath(){
+            return this.pklSchemaPath;
+        }
+
+        private String getPKLContext(){
+            return this.pklContext;
+        }
+    }
+    public static final List<PKLSchema> supportedPKLSchemas = Collections.unmodifiableList
+            (new ArrayList<PKLSchema>() {{ add( new PKLSchema("org/smpte_ra/schemas/st0429_8_2007/PKL/packingList_schema.xsd", "org.smpte_ra.schemas.st0429_8_2007.PKL"));
+                add( new PKLSchema("org/smpte_ra/schemas/st0429_8_2016/PKL/packingList_schema.xsd", "org.smpte_ra.schemas.st0429_8_2016.PKL"));}});
```

Comment:
Given that the namespace for 2016 PKL is "http://www.smpte-ra.org/schemas/2067-2/2016/PKL", should we consider renaming arguments "org/smpte_ra/schemas/st0429_8_2016/PKL/packingList_schema.xsd" and "org.smpte_ra.schemas.st0429_8_2016.PKL"?